qcodes/instrument/parameter.py

```diff
@@ -1287,6 +1265,58 @@ class DelegateParameter(Parameter):
     without changing those in the source parameter
     """
 
+    class _DelegateCache():
+        def __init__(self,
+                     source: _BaseParameter,
+                     parameter: _BaseParameter):
+            self._source = source
+            self._parameter = parameter
+
```

Comment:
until the snapshot_base improvement #1833 , delegate cache has to also have `_value` forwarded to `source.cache._value`, sorry :(

Reply:
:-( Okay